electron-rebuild fails on Eclipse CI on the Windows Node

Since the latest Theia upgrade and the switch to electron 15.3.6 we get build failures on the windows node:
See https://ci.eclipse.org/theia/job/Theia2/job/master/27/ 
The Github-Action build works however: https://github.com/eclipse-theia/theia-blueprint/runs/5042647096?check_suite_focus=true

<details>
  <summary>Full error log</summary>

```
theia-blueprint: $ theia rebuild:electron && yarn bundle
theia-blueprint: Module not found: find-git-repositories
theia-blueprint: Processed "native-keymap"
theia-blueprint: Processed "drivelist"
theia-blueprint: Processed "node-pty"
theia-blueprint: Processed "nsfw"
theia-blueprint: - Searching dependency tree
theia-blueprint: âœ– Rebuild Failed
theia-blueprint: [31m
theia-blueprint: An unhandled error occurred inside electron-rebuild[39m
theia-blueprint: [31mgyp info it worked if it ends with ok
theia-blueprint: gyp info using node-gyp@6.1.0
theia-blueprint: gyp info using node@12.20.0 | win32 | x64
theia-blueprint: gyp info find Python using Python version 3.9.1 found at "C:\Python39\python.exe"
theia-blueprint: gyp info find VS using VS2015 (14.0) found at:
theia-blueprint: gyp info find VS "C:\Program Files (x86)\Microsoft Visual Studio 14.0"
theia-blueprint: gyp info find VS run with --verbose for detailed information
theia-blueprint: gyp info spawn C:\Python39\python.exe
theia-blueprint: gyp info spawn args [
theia-blueprint: gyp info spawn args   'C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\node-gyp\\gyp\\gyp_main.py',
theia-blueprint: gyp info spawn args   'binding.gyp',
theia-blueprint: gyp info spawn args   '-f',
theia-blueprint: gyp info spawn args   'msvs',
theia-blueprint: gyp info spawn args   '-I',
theia-blueprint: gyp info spawn args   'C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\drivelist\\build\\config.gypi',
theia-blueprint: gyp info spawn args   '-I',
theia-blueprint: gyp info spawn args   'C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\node-gyp\\addon.gypi',
theia-blueprint: gyp info spawn args   '-I',
theia-blueprint: gyp info spawn args   'C:\\Users\\genie.theia\\.electron-gyp\\15.3.6\\include\\node\\common.gypi',
theia-blueprint: gyp info spawn args   '-Dlibrary=shared_library',
theia-blueprint: gyp info spawn args   '-Dvisibility=default',
theia-blueprint: gyp info spawn args   '-Dnode_root_dir=C:\\Users\\genie.theia\\.electron-gyp\\15.3.6',
theia-blueprint: gyp info spawn args   '-Dnode_gyp_dir=C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\node-gyp',
theia-blueprint: gyp info spawn args   '-Dnode_lib_file=C:\\\\Users\\\\genie.theia\\\\.electron-gyp\\\\15.3.6\\\\<(target_arch)\\\\node.lib',
theia-blueprint: gyp info spawn args   '-Dmodule_root_dir=C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\drivelist',
theia-blueprint: gyp info spawn args   '-Dnode_engine=v8',
theia-blueprint: gyp info spawn args   '--depth=.',
theia-blueprint: gyp info spawn args   '--no-parallel',
theia-blueprint: gyp info spawn args   '--generator-output',
theia-blueprint: gyp info spawn args   'C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\drivelist\\build',
theia-blueprint: gyp info spawn args   '-Goutput_dir=.'
theia-blueprint: gyp info spawn args ]
theia-blueprint: gyp info spawn C:\Program Files (x86)\MSBuild\14.0\bin\MSBuild.exe
theia-blueprint: gyp info spawn args [
theia-blueprint: gyp info spawn args   'build/binding.sln',
theia-blueprint: gyp info spawn args   '/clp:Verbosity=minimal',
theia-blueprint: gyp info spawn args   '/nologo',
theia-blueprint: gyp info spawn args   '/p:Configuration=Release;Platform=x64'
theia-blueprint: gyp info spawn args ]
theia-blueprint: Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
theia-blueprint:   drivelist.cpp
theia-blueprint:   device-descriptor.cpp
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1666): error C2059: syntax error: '' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1668): error C2238: unexpected token(s) preceding ';' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1684): error C2061: syntax error: identifier 'ResolveCallback' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(7341): error C2059: syntax error: '' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(8730): error C2061: syntax error: identifier 'HostImportModuleDynamicallyCallback' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1666): error C2059: syntax error: '' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1668): error C2238: unexpected token(s) preceding ';' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1684): error C2061: syntax error: identifier 'ResolveCallback' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(7341): error C2059: syntax error: '' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint:   list.cpp
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(8730): error C2061: syntax error: identifier 'HostImportModuleDynamicallyCallback' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint:   win_delay_load_hook.cc
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1666): error C2059: syntax error: '' (compiling source file ..\src\windows\list.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1668): error C2238: unexpected token(s) preceding ';' (compiling source file ..\src\windows\list.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1684): error C2061: syntax error: identifier 'ResolveCallback' (compiling source file ..\src\windows\list.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(7341): error C2059: syntax error: '' (compiling source file ..\src\windows\list.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(8730): error C2061: syntax error: identifier 'HostImportModuleDynamicallyCallback' (compiling source file ..\src\windows\list.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: gyp ERR! build error 
theia-blueprint: gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\MSBuild.exe` failed with exit code: 1
theia-blueprint: gyp ERR! stack     at ChildProcess.onExit (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\node-gyp\lib\build.js:194:23)
theia-blueprint: gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
theia-blueprint: gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
theia-blueprint: gyp ERR! System Windows_NT 10.0.19042
theia-blueprint: gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--target=15.3.6" "--arch=x64" "--dist-url=https://www.electronjs.org/headers" "--build-from-source"
theia-blueprint: gyp ERR! cwd C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist
theia-blueprint: gyp ERR! node -v v12.20.0
theia-blueprint: gyp ERR! node-gyp -v v6.1.0
theia-blueprint: gyp ERR! not ok 
theia-blueprint: Failed with exit code: 1
theia-blueprint: Error: gyp info it worked if it ends with ok
theia-blueprint: gyp info using node-gyp@6.1.0
theia-blueprint: gyp info using node@12.20.0 | win32 | x64
theia-blueprint: gyp info find Python using Python version 3.9.1 found at "C:\Python39\python.exe"
theia-blueprint: gyp info find VS using VS2015 (14.0) found at:
theia-blueprint: gyp info find VS "C:\Program Files (x86)\Microsoft Visual Studio 14.0"
theia-blueprint: gyp info find VS run with --verbose for detailed information
theia-blueprint: gyp info spawn C:\Python39\python.exe
theia-blueprint: gyp info spawn args [
theia-blueprint: gyp info spawn args   'C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\node-gyp\\gyp\\gyp_main.py',
theia-blueprint: gyp info spawn args   'binding.gyp',
theia-blueprint: gyp info spawn args   '-f',
theia-blueprint: gyp info spawn args   'msvs',
theia-blueprint: gyp info spawn args   '-I',
theia-blueprint: gyp info spawn args   'C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\drivelist\\build\\config.gypi',
theia-blueprint: gyp info spawn args   '-I',
theia-blueprint: gyp info spawn args   'C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\node-gyp\\addon.gypi',
theia-blueprint: gyp info spawn args   '-I',
theia-blueprint: gyp info spawn args   'C:\\Users\\genie.theia\\.electron-gyp\\15.3.6\\include\\node\\common.gypi',
theia-blueprint: gyp info spawn args   '-Dlibrary=shared_library',
theia-blueprint: gyp info spawn args   '-Dvisibility=default',
theia-blueprint: gyp info spawn args   '-Dnode_root_dir=C:\\Users\\genie.theia\\.electron-gyp\\15.3.6',
theia-blueprint: gyp info spawn args   '-Dnode_gyp_dir=C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\node-gyp',
theia-blueprint: gyp info spawn args   '-Dnode_lib_file=C:\\\\Users\\\\genie.theia\\\\.electron-gyp\\\\15.3.6\\\\<(target_arch)\\\\node.lib',
theia-blueprint: gyp info spawn args   '-Dmodule_root_dir=C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\drivelist',
theia-blueprint: gyp info spawn args   '-Dnode_engine=v8',
theia-blueprint: gyp info spawn args   '--depth=.',
theia-blueprint: gyp info spawn args   '--no-parallel',
theia-blueprint: gyp info spawn args   '--generator-output',
theia-blueprint: gyp info spawn args   'C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\drivelist\\build',
theia-blueprint: gyp info spawn args   '-Goutput_dir=.'
theia-blueprint: gyp info spawn args ]
theia-blueprint: gyp info spawn C:\Program Files (x86)\MSBuild\14.0\bin\MSBuild.exe
theia-blueprint: gyp info spawn args [
theia-blueprint: gyp info spawn args   'build/binding.sln',
theia-blueprint: gyp info spawn args   '/clp:Verbosity=minimal',
theia-blueprint: gyp info spawn args   '/nologo',
theia-blueprint: gyp info spawn args   '/p:Configuration=Release;Platform=x64'
theia-blueprint: gyp info spawn args ]
theia-blueprint: Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
theia-blueprint:   drivelist.cpp
theia-blueprint:   device-descriptor.cpp
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1666): error C2059: syntax error: '' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1668): error C2238: unexpected token(s) preceding ';' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1684): error C2061: syntax error: identifier 'ResolveCallback' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(7341): error C2059: syntax error: '' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(8730): error C2061: syntax error: identifier 'HostImportModuleDynamicallyCallback' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1666): error C2059: syntax error: '' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1668): error C2238: unexpected token(s) preceding ';' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1684): error C2061: syntax error: identifier 'ResolveCallback' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(7341): error C2059: syntax error: '' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint:   list.cpp
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(8730): error C2061: syntax error: identifier 'HostImportModuleDynamicallyCallback' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint:   win_delay_load_hook.cc
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1666): error C2059: syntax error: '' (compiling source file ..\src\windows\list.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1668): error C2238: unexpected token(s) preceding ';' (compiling source file ..\src\windows\list.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1684): error C2061: syntax error: identifier 'ResolveCallback' (compiling source file ..\src\windows\list.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(7341): error C2059: syntax error: '' (compiling source file ..\src\windows\list.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(8730): error C2061: syntax error: identifier 'HostImportModuleDynamicallyCallback' (compiling source file ..\src\windows\list.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: gyp ERR! build error 
theia-blueprint: gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\MSBuild.exe` failed with exit code: 1
theia-blueprint: gyp ERR! stack     at ChildProcess.onExit (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\node-gyp\lib\build.js:194:23)
theia-blueprint: gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
theia-blueprint: gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
theia-blueprint: gyp ERR! System Windows_NT 10.0.19042
theia-blueprint: gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\genie.theia\\jenkins_agent\\workspace\\Theia2_master\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--target=15.3.6" "--arch=x64" "--dist-url=https://www.electronjs.org/headers" "--build-from-source"
theia-blueprint: gyp ERR! cwd C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist
theia-blueprint: gyp ERR! node -v v12.20.0
theia-blueprint: gyp ERR! node-gyp -v v6.1.0
theia-blueprint: gyp ERR! not ok 
theia-blueprint: Failed with exit code: 1
theia-blueprint:     at SafeSubscriber._error (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\spawn-rx\lib\src\index.js:267:84)
theia-blueprint:     at SafeSubscriber.__tryOrUnsub (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\rxjs\internal\Subscriber.js:205:16)
theia-blueprint:     at SafeSubscriber.error (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\rxjs\internal\Subscriber.js:156:26)
theia-blueprint:     at Subscriber._error (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\rxjs\internal\Subscriber.js:92:26)
theia-blueprint:     at Subscriber.error (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\rxjs\internal\Subscriber.js:72:18)
theia-blueprint:     at MapSubscriber.Subscriber._error (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\rxjs\internal\Subscriber.js:92:26)
theia-blueprint:     at MapSubscriber.Subscriber.error (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\rxjs\internal\Subscriber.js:72:18)
theia-blueprint:     at SafeSubscriber._next (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\spawn-rx\lib\src\index.js:242:65)
theia-blueprint:     at SafeSubscriber.__tryOrUnsub (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\rxjs\internal\Subscriber.js:205:16)
theia-blueprint:     at SafeSubscriber.next (C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\rxjs\internal\Subscriber.js:143:22)[39m
theia-blueprint: Reverted "native-keymap"
theia-blueprint: Reverted "drivelist"
theia-blueprint: Reverted "nsfw"
theia-blueprint: Reverted "node-pty"
theia-blueprint: error Command failed with exit code 4294967295.
theia-blueprint: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
theia-blueprint: error Command failed with exit code 4294967295.
theia-blueprint: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run prepare exited 4294967295 in 'theia-blueprint'
error Command failed with exit code 4294967295.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

</details>

I think those syntax errors seem like the main issue
```
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1666): error C2059: syntax error: '' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1668): error C2238: unexpected token(s) preceding ';' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1684): error C2061: syntax error: identifier 'ResolveCallback' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(7341): error C2059: syntax error: '' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(8730): error C2061: syntax error: identifier 'HostImportModuleDynamicallyCallback' (compiling source file ..\src\drivelist.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1666): error C2059: syntax error: '' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1668): error C2238: unexpected token(s) preceding ';' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(1684): error C2061: syntax error: identifier 'ResolveCallback' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
theia-blueprint: c:\users\genie.theia\.electron-gyp\15.3.6\include\node\v8.h(7341): error C2059: syntax error: '' (compiling source file ..\src\device-descriptor.cpp) [C:\Users\genie.theia\jenkins_agent\workspace\Theia2_master\node_modules\drivelist\build\drivelist.vcxproj]
```

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -114,6 +114,7 @@ spec:
                     }
                     steps {
                         script {
+                            sh "npm config set msvs_version 2017"
                             buildInstaller()
                         }
                         stash name: 'win'

--- a/next/Jenkinsfile
+++ b/next/Jenkinsfile
@@ -90,6 +90,7 @@ spec:
                     }
                     steps {
                         script {
+                            sh "npm config set msvs_version 2017"
                             buildNext(true)
                         }
                     }

